### DISCUSSION TITLE
Mana Shield Hellfire bug

### DISCUSSION THREAD

**Discussion by noizerX** [diasurgical/DevilutionX#2343] [Q&A] [answered]
The bug of increasing Mana Shield level it becomes less efficient is still on in the currently DevilutionX Diablo 1 : Hellfire? I'm asking because I don't want to buy more than one Mana Shield book if the bug is still on.

If the bug is already fixed, how the reduction scales with slvl?

Thanks.

**Comment by qndel** [comment, marked as answer]
it doesn't, it's 0% for unlearned spell and 33% for any level above 0
